builder/tencentcloud/cvm/common.go

```diff
@@ -118,11 +128,20 @@ func NewCvmClient(secretId, secretKey, region string) (client *cvm.Client, err e
 }
 
 // NewVpcClient returns a new vpc client
-func NewVpcClient(secretId, secretKey, region string) (client *vpc.Client, err error) {
+func NewVpcClient(secretId, secretKey, region, endpoint string) (client *vpc.Client, err error) {
 	cpf := profile.NewClientProfile()
 	cpf.HttpProfile.ReqMethod = "POST"
 	cpf.HttpProfile.ReqTimeout = 300
 	cpf.Language = "en-US"
+	if endpoint != "" {
+		u, _ := url.Parse(endpoint)
```

Comment:
Same here, if `url.Parse` fails, the returned object will be nil, and the access to `u.Scheme` will then crash the plugin because of the nil dereference